src/lib/serializeMessageToSchemaRegistryAvro.test.js

```diff
@@ -75,3 +75,20 @@ test('Should serialize a big message to schema registry valid avro', () => {
   expect(schemaId).toBe(1);
   expect(decodedMessage).toEqual(message);
 });
+
+test('Should throw in attempt to serialize invalid message to schema registry avro', () => {
+  const registry = registryFixture();
+  // const avroType = schemaFixture();
+
+  const message = {
+    name: 55,
+    int: 'notAnInt' // error here as it should be an int.
+  };
+
+  expect(() =>
```

Comment:
Can you provide the actual error message in your test as well.  I want to make sure we can key off of it if I want to provide my own error messaging back to the ui that isnt the default format.

Reply:
Done, as per our conversation